browser/app/less/inc/alert.less

```diff
@@ -36,33 +31,32 @@
     }
 
     &.alert-info {
-        background: @blue;
+        background: @light-blue;
     }
 
-    @media(max-width: (@screen-xs-max)) {
+    @media (max-width: (@screen-xs-max)) {
         left: 20px;
         width: ~"calc(100% - 40px)";
         max-width: 100%;
     }
 
-    .progress {
-        margin: 10px 10px 8px 0;
-        height: 5px;
-        box-shadow: none;
-        border-radius: 1px;
-        background-color: @blue;
-        border-radius: 2px;
-        overflow: hidden;
-    }
-
-    .progress-bar {
-        box-shadow: none;
-        background-color: @white;
-        height: 100%;
-    }
-
     .close {
```

Comment:
It now happens in chrome, although slightly different. This is as of your latest update.
![screen shot 2017-07-31 at 5 20 21 pm](https://user-images.githubusercontent.com/3038254/28804006-62797274-7615-11e7-8dc3-97d8fe80e198.png)
It is however correctly centered in firefox now 😄